core/src/main/java/apoc/trigger/TriggerHandlerWrite.java

```diff
@@ -0,0 +1,153 @@
+        checkEnabled();
+        final HashMap<String, Object> previous = new HashMap<>();
+
+        withSystemDb(tx -> {
+            getTriggerNodes(databaseName, tx, triggerName)
+                    .forEachRemaining(node -> {
+                                previous.putAll(TriggerHandlerWrite.toTriggerInfo(node));
+                                node.delete();
+                            });
+            
+            setLastUpdate(databaseName, tx);
+            return null;
+        });
+
+        return previous;
+    }
+
+    public static Map<String, Object> updatePaused(String databaseName, String name, boolean paused) {
+        checkEnabled();
+        HashMap<String, Object> result = new HashMap<>();
+
+        withSystemDb(tx -> {
+            getTriggerNodes(databaseName, tx, name)
+                    .forEachRemaining(node -> {
+                        node.setProperty( SystemPropertyKeys.paused.name(), paused );
+                        result.putAll(TriggerHandlerWrite.toTriggerInfo(node));
+                    });
+
+            setLastUpdate(databaseName, tx);
+            return null;
+        });
+
+        return result;
+    }
+
+    public static Map<String, Object> dropAll(String databaseName) {
+        checkEnabled();
+        HashMap<String, Object> previous = new HashMap<>();
+
+        withSystemDb(tx -> {
+            getTriggerNodes(databaseName, tx)
+                    .forEachRemaining(node -> {
+                        String triggerName = (String) node.getProperty(SystemPropertyKeys.name.name());
+                        previous.put(triggerName, TriggerHandlerWrite.toTriggerInfo(node));
+                        node.delete();
+                    });
+            setLastUpdate(databaseName, tx);
+            return null;
+        });
+
+        return previous;
+    }
+
+    public static ResourceIterator<Node> getTriggerNodes(String databaseName, Transaction tx) {
+        return getTriggerNodes(databaseName, tx, null);
+    }
+    
+    public static ResourceIterator<Node> getTriggerNodes(String databaseName, Transaction tx, String name) {
+        final SystemLabels label = SystemLabels.ApocTrigger;
+        final String dbNameKey = SystemPropertyKeys.database.name();
+        if (name == null) {
+            return tx.findNodes(label, dbNameKey, databaseName);
+        }
+        return tx.findNodes(label, dbNameKey, databaseName,
+                SystemPropertyKeys.name.name(), name);
+    }
+
+    public static <T> T withSystemDb(Function<Transaction, T> action) {
+        try (Transaction tx = apocConfig().getSystemDb().beginTx()) {
+            T result = action.apply(tx);
+            tx.commit();
+            return result;
+        }
+    }
+
+    private static void setLastUpdate(String databaseName, Transaction tx) {
+        Node node = tx.findNode(SystemLabels.ApocTriggerMeta, SystemPropertyKeys.database.name(), databaseName);
```

Comment:
If we agree that new triggers should be decoupled from the old triggers (see https://github.com/neo4j-contrib/neo4j-apoc-procedures/pull/3183/files#r1005590912) then I think we should also decouple the node label being used to store trigger metadata.

Reply:
I've changed the caching logic. Now new and deprecated procedure use the same `TriggerHandler`, so we use the same system labels